src/sst/core/sst_mpi.h

```diff
@@ -0,0 +1,37 @@
+// Copyright 2009-2025 NTESS. Under the terms
+// of Contract DE-NA0003525 with NTESS, the U.S.
+// Government retains certain rights in this software.
+//
+// Copyright (c) 2009-2025, NTESS
+// All rights reserved.
+//
+// This file is part of the SST software package. For license
+// information, see the LICENSE file in the top level directory of the
+// distribution.
+
+#ifndef SST_CORE_SST_MPI_H
+#define SST_CORE_SST_MPI_H
+
+#include "sst_config.h"
```

Comment:
sst_config.h is not allowed to be included in header files and is required to be included first in all .cc files.  It's a quirk of the auto tools generating the file without guards for multiple inclusion.